deploy/10_osd-metrics-exporter.ClusterRole.yaml

```diff
@@ -0,0 +1,43 @@
+# this file contains the clusterroles needed for the core-functionality of the
+# operator.
+kind: ClusterRole
```

Comment:
Is there a reason the ClusteRole and the ClusterRoleBinding are not part of the olm-artifacts-template, but the Roles & RoleBinding are?

Reply:
Yes, the roles/rolebindings that are in the olm-artifacts-template are those that can not be deployed by olm alone, either because they are for a different namespace ( in this case it's for reading configmaps in `openshift-config`) or because they are for another serviceaccount ( allowing the `prometheus-k8s` sa to access the metrics in this namespace )